### PR TITLE
Remove $ from deps

### PR DESCRIPTION
**Pull request by jeetiss**

### CODE REVIEW

**Comment by nd0ut** [comment]
What happened to the branch name? :)

**Comment by jeetiss** [comment]
I used `$` symbol in it

File: src/files.js [hunk 1]
```diff
@@ -1,40 +1,7 @@
...
-      info = param[1]
-      param = param[0]
-    }
-    results.push(new converters[type](param, s, info).promise())
-  }
-  return results
-}
-
-export { fileFrom, filesFrom }
+export { fileFrom }
```
Comment (nd0ut) [inline src/files.js]: Why `filesFrom` was removed?

File: src/widget/tabs/camera-tab.js [hunk 2]
```diff
@@ -270,7 +270,7 @@ class CameraTab {
     return canvasToBlob(canvas, 'image/jpeg', 0.9, blob => {
       canvas.width = canvas.height = 1
       blob.name = 'camera.jpg'
-      this.dialogApi.addData('object', blob)
+      this.dialogApi.addData('object', [blob])
```
Comment (nd0ut) [inline src/widget/tabs/camera-tab.js]: Why `blob` is wrapped into array? Didn't noticed changes to the `addData`

Comment (jeetiss) [reply src/widget/tabs/camera-tab.js]: because `addData` accept array

**Pull request merged**